src/alto-ui/Aside/__tests__/Aside.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Aside from '../Aside';
+import CloseButton from '../../CloseButton';
+
+jest.mock('../../CloseButton', () => {
+  return () => {
+    return <div className="CloseButton" />;
+  };
+});
+
+describe('<Aside />', () => {
+
+  const getWrapper = props => {
+    console.log(props);
+    return mount(<Aside {...props} />);
+  };
+
+  it('should return null if show props is falsy', () => {
+    const props = {
+      show: false,
+    };
+    const wrapper = getWrapper(props);
+
+    expect(wrapper.find('.Aside__header')).toHaveLength(0);
```

Comment:
assertion should be only one and looks like:
```
expect(wrapper.find('aside')).toHaveLenght(0);
```

Reply:
Refactored